shared/common/src/commonMain/kotlin/com/tapptitude/tappchart/model/ValueLabel.kt

```diff
@@ -1,5 +1,4 @@
 package com.tapptitude.tappchart.model
 
-@JvmInline
-value class ValueLabel(val label: String)
+data class ValueLabel(val label: String, val yValue: Float)
```

Comment:
We need to avoid using data classes. See https://jakewharton.com/public-api-challenges-in-kotlin/

We should adopt the behaviour implemented in the Material library, where they implement each function (hashCode, copy, toString) by hand